docs/INSTALLATION.md

```diff
@@ -1,36 +1,30 @@
 # Installation
+You can download the latest Racket installer from [the official Racket website](https://download.racket-lang.org/).  Installers are available for Windows (64-bit and 32-bit), macOS (Apple Silicon and Intel) and Linux (64-bit).
 
-Please reference Racket's [Getting Started](http://docs.racket-lang.org/getting-started/) page for instructions on downloading and installing Racket.
+If you prefer to use a package manager, like `brew` on MacOS, `winget` on Windows or your Linux distribution's package manager, see [the Racket Wiki](https://github.com/racket/racket/wiki/Installing-Racket) for more detailed instructions.
 
-Alternative installation options and further notes for specific OSes follow.
+Please reference Racket's [Getting Started](http://docs.racket-lang.org/getting-started/) page for guidance materials.
 
-## macOS
+The [Racket Guide](https://docs.racket-lang.org/guide/intro.html) introduces you to interacting with Racket. 
```

Comment:
I'm not sure these links belong in this document, as they aren't about installing Racket. We have separate docs for learning resources.